runtime/vm/resolvesupport.cpp

```diff
@@ -1369,17 +1435,71 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 			lookupOptions |= J9_LOOK_CLCONSTRAINTS;
 		}
 		
-#if defined(J9VM_OPT_METHOD_HANDLE)
-		/*
-		 * Check for MH.invoke and MH.invokeExact.
-		 *
-		 * Methodrefs corresponding to those methods already have their methodIndex set to index into
-		 * cpClass->methodTypes. We resolve them by calling into MethodType.fromMethodDescriptorString()
-		 * and storing the result into the cpClass->methodTypes table.
-		 */
+
+#if defined(J9VM_OPT_METHOD_HANDLE) || defined(J9VM_OPT_OPENJDK_METHODHANDLE)
 		if ((NULL != cpShapeDescription)
 		&& (resolvedClass == J9VMJAVALANGINVOKEMETHODHANDLE(vm))
 		) {
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
```

Comment:
Maybe I'm looking for the wrong thing, but I'd expect the `invokeCache` to be filled in by the MH resolves.  Is that code still to come in a future PR?